resources/config.jsonld

```diff
@@ -0,0 +1,50 @@
+{
+  "@context": {
```

Comment:
Do we still support a regular json config file, or do we require users to update their config?

Reply:
user's will need to update their config.